packages/contracts/src/contracts/Settings.es

```diff
@@ -0,0 +1,5 @@
+{
+  // This boxes registers hold values used in order execution.
+  // Since this box is unspendable the settings are immutable and cannot be altered after the settings box has been minted.
```

Comment:
I believe we should think about ways to add governance mechanisms here, but it's good for testing now )

Reply:
Updated comment